internal/tests/ApplicationAccessGrantResource/application_access_grant_resource_test.go

```diff
@@ -0,0 +1,71 @@
+package ApplicationAccessGrantResource
+
+import (
+	. "axual.com/terraform-provider-axual/internal/tests"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestApplicationAccessGrantAutoApprovalResource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: GetProviderConfig(t).ProtoV6ProviderFactories,
+		ExternalProviders:        GetProviderConfig(t).ExternalProviders,
+
+		Steps: []resource.TestStep{
+			{
+				Config: GetProvider() + GetFile("axual_application_access_grant_auto_approval.tf"),
```

Comment:
Can we extract the config variable here and reuse it for cleanup, maybe in other places as well

Reply:
Sorry but I don't want to do it because often I want to comment out test steps and then change the test file in the cleanup. Currently it is very clear what is `.tf` file that is being applied, if we start abstracting them away to avoid duplication in only 2 places, it becomes more difficult to understand. Premature abstraction I would say.